### PR TITLE
[DOC] Added API policy to the README

### PR DESCRIPTION
**Pull request by ericmjl**
Closes issue #615.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -325,6 +325,23 @@ any feature requests will be prioritized according to
...
+(and other dataframe APIs),
+but will never fix or replace them.
+
+Undesirable ``pandas`` behaviour should be reported upstream
+in the ``pandas`` repository.
+If at some point the pandas devs
+decide to take something from pyjanitor
+and internalize it as part of the official pandas API,
+then we will deprecate it from pyjanitor,
+while acknowledge the original contributors' contribution
```
Comment (hectormz) [inline README.rst]: would change to:

while acknowledg**ing** the original contributors' contribution

Comment (hectormz) [inline README.rst]: Do you have thoughts on converting the rest of pyjanitor and pandas to ``pyjanitor`` and ``pandas`` ?

File: README.rst [hunk 2]
```diff
@@ -325,6 +325,23 @@ any feature requests will be prioritized according to
...
 
+API Policy
+~~~~~~~~~~
+
+``pyjanitor`` only extends or aliases the pandas API
+(and other dataframe APIs),
+but will never fix or replace them.
+
+Undesirable ``pandas`` behaviour should be reported upstream
+in the ``pandas`` repository.
```
Comment (hectormz) [inline README.rst]: Maybe we can add a hyperlink to `pandas` [issues page](https://github.com/pandas-dev/pandas/issues)?

File: README.rst (hunk 1 above)
Comment (ericmjl) [reply README.rst]: Good catch, thanks!

Comment (ericmjl) [reply README.rst]: Good catch too - changing!

File: README.rst (hunk 2 above)
Comment (ericmjl) [reply README.rst]: Also a very good idea. Thanks @hectormz!

**Comment by ericmjl** [comment]
@hectormz I think I’ve addressed your comments. Let me know what you think!

File: README.rst [hunk 3]
```diff
@@ -325,6 +325,24 @@ any feature requests will be prioritized according to
 what maintainers encounter as a need in our day-to-day jobs.
 Please temper expectations accordingly.
 
+API Policy
+~~~~~~~~~~
+
+``pyjanitor`` only extends or aliases the pandas API
```
Comment (hectormz) [inline README.rst]: One last change to ``pandas`` here, and it's good to go

**Comment by hectormz** [comment]
@ericmjl one last small change and I think it's ready to merge!

**Comment by ericmjl** [comment]
@hectormz looks like we're good to go!

**Comment by hectormz** [comment]
@ericmjl  agreed!

**Pull request merged**